packages/atomic/src/components/atomic-search-interface/atomic-aria-live.tsx

```diff
@@ -0,0 +1,71 @@
+import {
+  buildQuerySummary,
+  buildSearchStatus,
+  QuerySummary,
+  QuerySummaryState,
+  SearchStatus,
+  SearchStatusState,
+} from '@coveo/headless';
+import {Component, h, Host, State} from '@stencil/core';
+import {
+  Bindings,
+  BindStateToController,
+  InitializeBindings,
+} from '../../utils/initialization-utils';
+
+/**
+ * The `atomic-aria-live` component notifies screen readers of changes in the search interface.
+ * @internal
+ */
+@Component({
+  tag: 'atomic-aria-live',
+  shadow: false,
+})
+export class AtomicAriaLive {
+  @InitializeBindings() public bindings!: Bindings;
+  @State() public error!: Error;
+
+  protected searchStatus!: SearchStatus;
+  @BindStateToController('searchStatus')
+  @State()
+  private searchStatusState!: SearchStatusState;
+  protected querySummary!: QuerySummary;
+  @BindStateToController('querySummary')
+  @State()
+  private querySummaryState!: QuerySummaryState;
+
+  private get searchSummary() {
+    if (!this.searchStatusState.firstSearchExecuted) {
+      return '';
+    }
+    if (this.searchStatusState.isLoading) {
+      return this.bindings.i18n.t('loading-results');
+    }
+    if (!this.searchStatusState.hasResults) {
+      return this.bindings.i18n.t('no-results');
```

Comment:
When no results are caused by an error due to the connection, the token etc. what would happen?
Should Atomic's aria-live announce a more descriptive message in these situations?

Reply:
Hum good point ... 🤔 Do we handle that in JSUI ?

Reply:
Yep, there is a separate [message](https://github.com/coveo/search-ui/blob/master/src/ui/AriaLive/AriaLive.ts#L39) for errors.

Reply:
Good point, I'll handle that.